src/imitation/algorithms/base.py

```diff
@@ -181,6 +182,14 @@ def set_demonstrations(self, demonstrations: AnyTransitions) -> None:
     def policy(self) -> policies.BasePolicy:
         """Returns a policy imitating the demonstration data."""
 
+    def save_policy(self, policy_path: types.AnyPath) -> None:
+        """Save policy to a path. Can be reloaded by `.reconstruct_policy()`.
```

Comment:
Where is reconstruct_policy defined? All I see in our codebase is a module-level method in the `bc` package.

Reply:
Oh, good point. save_policy was originally there too, but I wanted to move it to `BaseImitationAlgorithm` so that `sqil` could use it. But since they reference each other it makes sense to move `reconstruct_policy` there too, which I've done now.

Reply:
I've also now moved save_policy again fyi.

Reply:
OK to leave `reconstruct_policy` where it is for now, it's not needed anywhere else right now anyway.